### PR TITLE
PR: Allow leading indents in cells

### LINKED ISSUE #247: Remove from warning that future versions of Spyder will give an error when using non-Python syntax
Currently we're showing this warning

immagine

I think we should remove its last sentence and perhaps reword its last part. That's because it seems a lot of people depends on this feature and forcing them to use `ipy` files would be a nuisance.

@impact27, what do you think?

### PR DESCRIPTION
**Pull request by impact27**
Fixes #247

### CODE REVIEW

File: spyder_kernels/customize/spydercustomize.py [hunk 1]
```diff
@@ -30,7 +31,8 @@
 from spyder_kernels.customize.umr import UserModuleReloader
 
 if not PY2:
```
Comment (ccordoba12) [inline spyder_kernels/customize/spydercustomize.py:33]: The `inputtransformer2` module was introduced in IPython 7, so this condition needs to be changed in the same way as I did in https://github.com/spyder-ide/spyder/pull/13721

Comment (ccordoba12) [reply spyder_kernels/customize/spydercustomize.py:33]: Probably the `if`s you modified below need to be adjusted in the same way, but I'm not really sure.

**Review by ccordoba12 — COMMENTED** [review]
@impact27, I left a minor comment for you, otherwise looks good to me.

Could you also add a test for this? Thanks!

**Comment by impact27** [comment]
Fixes https://github.com/spyder-ide/spyder-kernels/issues/247

File: spyder_kernels/customize/spydercustomize.py (hunk 1 above)
Comment (impact27) [reply spyder_kernels/customize/spydercustomize.py]: should be fixed

**Comment by impact27** [comment]
Test implemented in https://github.com/spyder-ide/spyder/pull/13852

File: spyder_kernels/customize/spydercustomize.py [hunk 2]
```diff
@@ -442,9 +458,8 @@ def exec_code(code, filename, ns_globals, ns_locals=None, post_mortem=False):
                             "\nWARNING: This is not valid Python code. "
                             "If you want to use IPython magics, "
                             "flexible indentation, and prompt removal, "
-                            "please save this file with the .ipy extension. "
-                            "This will be an error in a future version of "
-                            "Spyder.\n")
+                            "we recommand that you save this file with the "
```
Comment (ccordoba12) [inline spyder_kernels/customize/spydercustomize.py:461]: recommand -> recommend

**Review by ccordoba12 — COMMENTED** [review]
I left a small suggestion, otherwise looks good.

File: spyder_kernels/customize/spydercustomize.py [hunk 3]
```diff
@@ -22,6 +22,8 @@
 import logging
 import cmd
 
+from distutils.version import LooseVersion
```
Comment (ccordoba12) [inline spyder_kernels/customize/spydercustomize.py:25]: `distutils` is part of the standard library, so it needs to go in the previous section of imports. Could you also organize them in alphabetical order as part of this PR? Thanks!

**Review by ccordoba12 — COMMENTED** [review]
Sorry @impact27, i missed another small thing.

**Review by ccordoba12 — APPROVED** [review]
Thanks @impact27!

**Pull request merged**